### PR TITLE
Implement Healthcare URL Mapping for VA.gov

### PR DESCRIPTION
**Pull request by ncksllvn**
These changes were generated by the script in department-of-veterans-affairs/vets.gov-utility/pull/21 based on the [health-care tab of this spreadsheet](https://docs.google.com/spreadsheets/d/1YucOtYihweNPI1gYtCgdsinKQmH4DMuSgPaSgJMfyYk/edit#gid=0).

Ultimately, the changes consist of:
- Renaming and moving the markdown files in `va-gov/pages`
- Replacing URLs in all files located in the `va-gov`, `src`, and `script` directories with the update VA.gov URl

An important note here is that because code (JS, JSX, JSON) was changed (specifically, app entry points) the regular `vets-website/content/` directory of this branch (`brand-conslidation`) is essentially broken.

Resolves department-of-veterans-affairs/vets.gov-team/issues/12368

### CODE REVIEW

**Comment by ncksllvn** [comment]
@bethpotts @minafarzad I'm guessing that the URLs in the navigation need to be updated as well. Should that be done in the PR? I'm hesitant because that will need to be done manually, since write the HREFs are just a `#` so the script doesn't know to replace them.

**Comment by accbjt** [comment]
@ncksllvn since this is breaking changes in the original content folder for vets.gov. What do we need to do to fix this, so we can eventually merge to master.

**Comment by bethpotts** [comment]
@ncksllvn Yes, the urls for the links down the middle of the health care hub page need to be added.

File: script/build.js [hunk 1]
```diff
@@ -346,42 +346,42 @@ smith.use(collections({
     }
```
[the remark this answers is not in the document]
Comment (accbjt) [reply script/build.js]: @ncksllvn Actually we won't need to add this condition in anymore after our discussion today.

**Pull request merged**